### ISSUE TITLE
Not able to reproduce the result with the same settings in the paper

### ISSUE THREAD

**Issue by MuscleGood** [putshua/SNN-conversion-QCFS#8]
Hi, with the same settings (all the hyperparameters) I still can't reproduce the result as reported in the paper.
In the exp on VGG on CIFAR10, when I set the l=4, the best acc of ann can't reach the result in the paper (0.938, 0.955 in the paper), and when I set l=32, the acc of snn at T=4 is 0.879 (0.939 in the paper).
And i'd like to know whether you add the autoAugment to the CIFAR100.

Thank you for providing the code, expecting for your reply, thank you in advance!

**Comment by putshua** [comment]
I have added the autoaugment  to CIFAR100 (The CIFAR10Policy).

**Comment by putshua** [comment]
For hyperparameters, please carefully refer to the appendix of the paper and do not use the deflaut settings in this repo. For example, the training epochs should be set to 300 and initial learning rate for CIFAR100 is 0.02. If you still cannot reproduce the same results, you can email me the logs, settings and your results so that I may find out the problem. Thanks.

**Comment by MuscleGood** [comment]
> I have added the autoaugment to CIFAR100 (The CIFAR10Policy).

Thanks for reply! But I found there's no CIFAR10Policy in 'Preprocess/getdataloader.py' , so I wonder if you forgot to update the correct version~

`def GetCifar100(batchsize):
    trans_t = transforms.Compose([transforms.RandomCrop(32, padding=4),
                                  transforms.RandomHorizontalFlip(),
                                  transforms.ToTensor(),
                                  transforms.Normalize(mean=[n/255. for n in [129.3, 124.1, 112.4]], std=[n/255. for n in [68.2,  65.4,  70.4]]),
                                  Cutout(n_holes=1, length=16)
                                  ])
    trans = transforms.Compose([transforms.ToTensor(), transforms.Normalize(mean=[n/255. for n in [129.3, 124.1, 112.4]], std=[n/255. for n in [68.2,  65.4,  70.4]])])

    train_data = datasets.CIFAR100(DIR['CIFAR100'], train=True, transform=trans_t, download=True)
    test_data = datasets.CIFAR100(DIR['CIFAR100'], train=False, transform=trans, download=True) 
    train_dataloader = DataLoader(train_data, batch_size=batchsize, shuffle=True, num_workers=8, pin_memory=True)
    test_dataloader = DataLoader(test_data, batch_size=batchsize, shuffle=False, num_workers=4, pin_memory=True)
    return train_dataloader, test_dataloader`

**Comment by putshua** [comment]
Thanks, I think this has little effect on the final results based on my previous experiments, so I will keep it.

**Comment by MuscleGood** [comment]
OK I got it, thank you! I'll check out whether I missed anything in reproducing the results.

**Comment by MuscleGood** [comment]
I'd like to know whether the seed will influence the results, I notice that the seed is set to 42 for all the models in default.

**Comment by MuscleGood** [comment]
I still can't reproduce the results in the paper, may I send you my logs, settings and results? And what file format shoud I use for the logs for your convenience?

**Comment by putshua** [comment]
Pretrained model updated.

**Comment by MuscleGood** [comment]
Thank you!

**Comment by putshua** [comment]
I have updated a new version in a new repo at https://github.com/putshua/ANN_SNN_QCFS. The biggest unreproducable bug is fixed. Compatible with old version. Please switch to the new version.
